src/up.js

```diff
@@ -66,6 +66,31 @@ module.exports = function up() {
           type: 'int',
           default: 3000,
         })
+        .option('action-param', {
```

Comment:
The code below is doing the same (conceptually) as the code here: https://github.com/adobe/helix-cli/blob/07ba48b0b9f8fdaf92c8eed7e7ada047f15676e3/src/deploy.js#L99-L107

In `hlx deploy --default key1 value1 --default key2 value2` is the same as `hlx up --action-param key1 value1 key2 value2`. I'd suggest we unify these two parameters by
- [ ] moving the code from `hlx deploy` into a `yargs-params.js` file
- [ ] adding `action-param` as an alias to defaults
- [ ] using the `yargs-params.js` function in both `hlx up` and `hlx deploy`

Reply:
I'm not sure this should be treated the same. you have different requirements, eg. for `HTTP_TIMEOUT` on your local dev instance than on production. since you can add the params to you `.env` files, this might add unexpected params during deployment.

we could rename it to `--dev-default` to align it closer to the deploy arguments.

Reply:
good idea @trieloff i'll do that. Do you think I should keep the approach from hlx up for checking params; I think it's more flexible to do 
--default KV KV  instead of --default KV --default KV, also the code in deploy assumes string values, which will break if you supply HTTP_TIMEOUT 2000; i'll just check for string

@tripodsan i'll keep your naming convention; I will keep --defualt as the primary; and then alias --dev-default.

Reply:
@MarquiseRosier I would not use `--default` for `hlx up`. As I said, it causes confusion when setting the parameters in `.env`.

Reply:
This is true. Hmmm I definitely want to make a yargs-params.js for this code; So i'm going to put two parameters in commonArgs(yargs, name, describe, alias, type, default) so we can reuse logic here

Reply:
> I think it's more flexible to do `--default KV KV` instead of `--default KV --default KV`

That might be, but I'd take the default that is built in with yargs.